meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -11,7 +11,12 @@ while [ ! -L "$DATA_PARTITION" ]; do
 		sleep 1 && end=$((end + 1))
 	else
 		echo "[ERROR] Timeout while waiting for filesystem label to appear."
-		return 1
+		echo "[ERROR] Something bad happened. Assuming storage corruption and attempting fsck + reboot."
+		fdisk -l | grep '^/dev' | grep -i 'FAT' |  cut -d' ' -f1 | while read line ; do fsck.fat -av $line; done
```

Comment:
Do we depend on util-linux for this recipe?

Reply:
I'll add util-linux as a dependency.